arm-scheduler/2014-08-01-preview/swagger/scheduler.json

```diff
@@ -39,7 +39,8 @@
                             "$ref": "#/definitions/JobCollectionListResult"
                         }
                     }
-                }
+                },
+                "x-ms-pageable": true
```

Comment:
This extension has changed. Should be: "x-ms-pageable": { "nextLinkValue" : "nextLink }

See other specs as a reference.